Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job stats:
job               count
--------------  -------
Final_plotting        1
all                   1
total                 2

Resources before job selection: {'_cores': 1, '_nodes': 9223372036854775807}
Ready jobs (1)
Select jobs to execute...
Using greedy selector because only single job has to be scheduled.
Selected jobs (1)
Resources after job selection: {'_cores': 0, '_nodes': 9223372036854775806}

[Sun Oct 27 22:22:36 2024]
rule Final_plotting:
    input: outp_file/mearged_plotting.tsv
    output: outp_file/box_plot.png
    jobid: 4
    reason: Missing output files: outp_file/box_plot.png
    resources: tmpdir=/tmp


        Rscript --vanilla ploting_data.R outp_file/mearged_plotting.tsv outp_file/box_plot.png
        
Full Traceback (most recent call last):
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2656, in run_wrapper
    run(
  File "/root/Project/Tcga_analysis/Saftey_folder/Main_file.smk", line 117, in __rule_Final_plotting
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/shell.py", line 294, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  
        Rscript --vanilla ploting_data.R outp_file/mearged_plotting.tsv outp_file/box_plot.png' returned non-zero exit status 1.

[Sun Oct 27 22:22:37 2024]
Error in rule Final_plotting:
    jobid: 4
    input: outp_file/mearged_plotting.tsv
    output: outp_file/box_plot.png
    shell:
        
        Rscript --vanilla ploting_data.R outp_file/mearged_plotting.tsv outp_file/box_plot.png
        
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2656, in run_wrapper
    run(
  File "/root/Project/Tcga_analysis/Saftey_folder/Main_file.smk", line 117, in __rule_Final_plotting
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/shell.py", line 294, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  
        Rscript --vanilla ploting_data.R outp_file/mearged_plotting.tsv outp_file/box_plot.png' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 715, in _callback
    raise ex
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 699, in cached_or_run
    run_func(*args)
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/site-packages/snakemake/executors/__init__.py", line 2692, in run_wrapper
    raise RuleException(
snakemake.exceptions.RuleException: CalledProcessError in file /root/Project/Tcga_analysis/Saftey_folder/Main_file.smk, line 60:
Command 'set -euo pipefail;  
        Rscript --vanilla ploting_data.R outp_file/mearged_plotting.tsv outp_file/box_plot.png' returned non-zero exit status 1.
  File "/root/Project/Tcga_analysis/Saftey_folder/Main_file.smk", line 60, in __rule_Final_plotting

RuleException:
CalledProcessError in file /root/Project/Tcga_analysis/Saftey_folder/Main_file.smk, line 60:
Command 'set -euo pipefail;  
        Rscript --vanilla ploting_data.R outp_file/mearged_plotting.tsv outp_file/box_plot.png' returned non-zero exit status 1.
  File "/root/Project/Tcga_analysis/Saftey_folder/Main_file.smk", line 60, in __rule_Final_plotting
  File "/root/miniconda3/envs/snakemake_env/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-10-27T222235.243612.snakemake.log
unlocking
removing lock
removing lock
removed all locks
